lib/Runtime/Library/JavascriptFunction.cpp

```diff
@@ -2793,6 +2793,10 @@ void __cdecl _alloca_probe_16()
             {
                 *value = scriptContext->GetLibrary()->GetNull();
             }
+            else
```

Comment:
@agarwal-sandeep not sure if I understood the question. Do you mean we have to return nullValue instead of marshalling when the contexts don't match ?